### PR TITLE
#104 - Added styles to Footer section

### PR DESCRIPTION
**Pull request by Hanna-N9**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -29,13 +29,55 @@ function App() {
...
+                <img
+                  className="bg-white rounded-full h-10"
+                  src="/images/insta.png"
+                  alt="instagram's icon"
+                />
+              </a>
+            </div>
+          </div>
+          <div class="mt-5">
+            <h2>Made with 🤝 by CodeDay Labs</h2>
```
Comment (mikelbrierly) [inline client/src/App.js:68]: Generally we want to reserver the heading tags for headings of the page. In this case we are in a footer, so there shouldn't be any headings inside this element. We should just stick to a `<p>` tag with a tailwind class for text-sizing here in the footer

File: client/src/App.js [hunk 2]
```diff
@@ -29,13 +29,55 @@ function App() {
...
         {/* <img src={logo} className="App-logo" alt="logo" /> */}
-
-         main
-        <p class="heading-font">{ !data ? "Loading..." : data}</p>{" "}
-
+        main
+        <p class="heading-font">{!data ? "Loading..." : data}</p>{" "}
         {/* If no data message, then it will load otherwise, there is a data message */}
+        <footer className="fixed bg-gray-800 w-full bottom-0 items-center flex flex-full flex-col py-20">
+          <div class="flex space-x-10 ...">
```
Comment (mikelbrierly) [inline client/src/App.js:37]: What does `...` do here?

**Comment by mikelbrierly** [comment]
This is awesome so far Hanna!! Your use of Tailwind is fantastic! And it looks great. Just a couple changes and we can merge it in! 

One thing as a sidenote - to go the extra mile with a PR for front-end work, it's really nice to add a screenshot or a gif of what you changed on the site so the reviewers can visualize your changes quickly without having to pull the branch down

File: client/src/App.js (hunk 2 above)
Comment (Hanna-N9) [reply client/src/App.js:37]: The three dots do nothing. Do you want me to add address for each media too or only for line 37.

Comment (mikelbrierly) [reply client/src/App.js:37]: Just line 37!

**Comment by Hanna-N9** [comment]
This is what it looks like:

6FB13932-525E-425F-9E85-C1FFF385EF54_1_105_c

**Review by mikelbrierly — APPROVED** [review]

**Pull request merged**